mix.exs

```diff
@@ -23,13 +23,14 @@ defmodule Cogctl.Mixfile do
       # We override here because of a conflict in rebar. Spanner brings in emqtt which includes
       # rebar as a dep.
       {:getopt, github: "jcomellas/getopt", tag: "v0.8.2", override: true},
-      {:ibrowse, "~> 4.2.2"},
+      {:ibrowse, "~> 4.2.2", override: true},
```

Comment:
Why is this overridden? It'd be a good idea to add a comment explaining why, as well as what it's being overridden from.